paimon-spark/paimon-spark-common/src/main/java/org/apache/paimon/spark/SparkConnectorOptions.java

```diff
@@ -24,6 +24,13 @@
 
 /** Options for spark connector. */
 public class SparkConnectorOptions {
+
+    public static final ConfigOption<Boolean> INNER_SESSION_CATALOG =
+            key("catalog.inner-session-catalog")
```

Comment:
`delegate-built-in-session-catalog`?
Default is true.

Reply:
`catalog.delegate-built-in-session-catalog`: true means use the original spark session catalog.